cms/envs/common.py

```diff
@@ -1255,3 +1255,7 @@
 ############## Settings for the Enterprise App ######################
 
 ENTERPRISE_ENROLLMENT_API_URL = LMS_ROOT_URL + "/api/enrollment/v1/"
+
+############## Settings for the Discovery App ######################
+
+COURSE_CATALOG_API_URL = None
```

Comment:
For any new settings we add, we must also create a corresponding PR in https://github.com/edx/configuration